packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/vmware-cloud-director/translations/Messages_fr_FR.json

```
@@ -1,12 +1,15 @@
 {
-  "dedicatedCloud_vmware_cloud_director_title": "Migration vers VCD",
+  "dedicatedCloud_vmware_cloud_director_title": "Migration vers l'offre Managed VCD",
   "dedicatedCloud_vmware_cloud_director_offer_detail": "Descriptif de l'offre VCD",
+  "dedicatedCloud_vmware_cloud_director_offer_terms_and_conditions": "Conditions particulières du service",
   "dedicatedCloud_vmware_cloud_director_new": "New",
-  "dedicatedCloud_vmware_cloud_director_subscription": "Souscription",
-  "dedicatedCloud_vmware_cloud_director_announcement_message": "Message d'annonce",
+  "dedicatedCloud_vmware_cloud_director_migration": "Migration",
+  "dedicatedCloud_vmware_cloud_director_migration_confirmed": "Migration confirmée",
+  "dedicatedCloud_vmware_cloud_director_migration_conf": "Migration confirmée",
```

```suggestion
  "dedicatedCloud_vmware_cloud_director_migration_confirmed": "Migration confirmée",
```
I don't know if it's a mistake.

---

It is. Good catch. Fixed !